cypress/fixtures/objects/validationObjects/accessibility.json

```diff
@@ -153,12 +166,13 @@
         "method": "accessibility.closedCaptionsSettings",
         "data": [
             {
-                "type": "fixture",
+                "type": "undefined",
                 "validations": [
                     {
-                        "mode": "staticContentValidation",
-                        "type": "ACCESSIBILITY_FONTEDGE_NONE",
-                        "description": "Validation of the accessibility fontEdge none"
+                        "field": "result.styles.fontEdge",
+                        "mode": "",
+                        "type": "",
+                        "description": "Validation of the accessibility fontEdge undefined"
                     }
                 ]
             }
```

Comment:
to maintain consistency, remove **mode** and **type** from all json objects if they are not used